QuestionScreen.md

```diff
@@ -1,6 +1,47 @@
-# QuestionScreen
+# Introducing the screens concept
+
+From now on, the screens concept will be around when talking about the interaction system we have designed using Botman. And, precisely because we are using Botman, a screen is an special type of question, extending the Botman question, designed to wrap a field question and control the interaction of the user with the question.
+
+As we think of this screens concept, it comes to mind that it is not specific for the implementation of pagination, we already have implementend some interactive features that work as separate from the process of actually answering a question, and that is absolutely right. It just happens that this is when we actually need it, and it's a must have in order to keep consistency, simplicity and logic.
+
+As a consequence of this implementation, some refactoring (not reinventing the wheel here, keep calm) is expected. Things like "Show more info", displaying errors and showing hints are going to be lifted from the conversation level to the screen level.
```

Comment:
@leninpaulino this seems to speak in future tense. Can you clarify if that was the intent? Thanks

Reply:
oh, copied and pasted from old notes, sorry. fixing it now